benchmarking/smartseq2/merge_expression_matrix/run_merge_quants.wdl

```diff
@@ -0,0 +1,38 @@
+task mergeTPM{
+  String target_dir
+  String run_name
+  String output_name
+  String uuid
+  String value_name
+  command{
+    gsutil cp  ${target_dir}/merge_table_quants.py ./   
```

Comment:
Should this script go in the docker container?

Reply:
yes, it is possible. I think eventually, after we've done the benchmarking implementation,  we will put these scripts into a docker.  Currently, I just try to avoid rebuild a new docker images every time if there is a little changes in scripts or adding new scripts.